pegasus/sites.v3/code.org/views/csp_information.md.erb

```diff
@@ -0,0 +1,25 @@
+<% facebook = {:u=>'https://youtu.be/DMr1iFYacGQ'} %>
+<% twitter = {:url=>'https://youtu.be/DMr1iFYacGQ', :related=>'codeorg', :text=>"Computer Science Principles. (Thanks #{get_random_donor_twitter} for supporting @codeorg)"} %>
+
+<div class="col-40" style="float:left; padding:10px;">
+
+  <%=view :display_video_thumbnail, id: "CSPrinciples", video_code: "DMr1iFYacGQ", play_button: 'center', facebook: facebook, twitter: twitter, letterbox: 'false' %>
+
+</div>
+
+[col-60]
+
+Designed for 9 - 12 grade students, CS Principles introduces students to the foundational concepts of computer science and challenges them to explore how computing and technology can impact the world. This year-long course can be taught as an introductory course and as an AP course - no prerequisites required for students or teachers new to computer science! CS Principles complements CS Discoveries with a deeper focus on concepts such as how the internet works and the societal impacts of computer science.
+
+<% facebook = {:u=>'https://youtu.be/DMr1iFYacGQ'} %>
+<% twitter = {:url=>'https://youtu.be/DMr1iFYacGQ', :related=>'codeorg', :text=>"Computer Science Principles. (Thanks #{get_random_donor_twitter} for supporting @codeorg)"} %>
+
+[<button>Learn about CS Principles</button>](/educate/csp) &nbsp;&nbsp;[<button>Learn about professional learning</button>](/educate/professional-learning/middle-high)
```

Comment:
is it intentional that you have a breaking space there, before the two non-breaking spaces?

Reply:
I'm not sure, but it turns out that one `&nbsp;` does the job, so I'll clean this up a little.

Reply:
my assumption is that the presence of a breaking space means the browser will happily wrap this line on that space if necessary, rendering the non-breaking spaces pointless. Am I misunderstanding something?

Reply:
Ah, in this case it produces a wider space than when it's not there.  I'm not sure of a better way to do this in markdown alone?

Reply:
what's your specific goal here?

Reply:
Just creating a bit more space between those two buttons than the default.

Reply:
ahhh, gotcha. I'd recommend using CSS for that, rather than markup